modules/ftrackIdSystem.js

```diff
@@ -102,9 +103,9 @@ export const ftrackIdSubmodule = {
         }
 
         if (config.params && config.params.url) {
-          var ftrackScript = document.createElement('script');
-          ftrackScript.setAttribute('src', config.params.url);
-          window.document.body.appendChild(ftrackScript);
+          var ftrackOnload = function (evt) {}
```

Comment:
I don't think this is necessary? loadExternalScript will attempt to call it only if it's defined.

Reply:
Thanks @dgirardi . You are correct, I removed this condition as well as the 3rd and 4th parameters in our loadExternalScript reference as well since we aren't utilizing them.